common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
What do I get if I send `GetBanHistory` with no username? All users? That seems like it would be a good base behavior - in the future we will need to add more specific querying (ip, client id, etc)